### PR TITLE
switch to http api

### PR DESCRIPTION
**Pull request by trel**
do not merge until HTTP API 0.4.0 is released.

### CODE REVIEW

**Comment by trel** [comment]
HTTP API 0.4.0 is out - I think this is ready for eyeballs and any testing by others.

**Review by korydraughn — APPROVED** [review]
Nothing unusual sticks out to me.

Had one suggestion, but feel free to ignore it if the changes work and you're happy with it.

File: docker-compose.yml [hunk 1]
```diff
@@ -54,30 +54,20 @@ services:
...
-            - irods-client-rest-cpp
+            irods-catalog-provider:
+                condition: service_healthy
 
     irods-client-zmt:
         build:
             context: irods_client_zone_management_tool
         environment:
-            - REACT_APP_REST_API_URL=http://localhost/irods-rest/0.9.3
+            - REACT_APP_HTTP_API_URL=http://localhost/irods-http-api/0.4.0
```
[the remark this answers is not in the document]
Comment (trel) [reply docker-compose.yml:70]: interesting.   might have to think about that... as this repo is for people 'learning' how our stuff fits together.

**Comment by trel** [comment]
correct - awaiting ZMT release first.

**Pull request merged**